configs/components/_base-ruby-augeas.rb

```diff
@@ -3,20 +3,15 @@
 # load it with instance_eval. See ruby-x.y-augeas.rb configs.
 #
 
-pkg.add_source("file://resources/patches/augeas/ruby-augeas-0.5.0-patch_c_extension.patch")
-
-# We can remove the below patch after https://github.com/hercules-team/ruby-augeas/pull/17 is merged.
-pkg.add_source("file://resources/patches/augeas/ruby-augeas-0.5.0-patch_remove_unused_parameter.patch")
-
 # These can be overridden by the including component.
 ruby_version ||= settings[:ruby_version]
 host_ruby ||= settings[:host_ruby]
 ruby_dir ||= settings[:ruby_dir]
 ruby_bindir ||= settings[:ruby_bindir]
 
-pkg.version "0.5.0"
-pkg.md5sum "a132eace43ce13ccd059e22c0b1188ac"
-pkg.url "http://download.augeas.net/ruby/ruby-augeas-#{pkg.get_version}.tgz"
+pkg.version "0.6.0"
+pkg.md5sum "3c2a13b748300b5a984bab9a30e74d0f"
+pkg.url "https://github.com/hercules-team/ruby-augeas/releases/download/release-#{pkg.get_version}/ruby-augeas-#{pkg.get_version}.tgz"
```

Comment:
I'm thinking we should only update puppet8 given the risk of regression in puppet7